gradle-sls-packaging/src/main/groovy/com/palantir/gradle/dist/RecommendedProductDependencies.java

```diff
@@ -0,0 +1,25 @@
+package com.palantir.gradle.dist;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import org.immutables.value.Value;
+
+import java.util.Set;
+
+@Value.Immutable
+@JsonSerialize(as = ImmutableRecommendedProductDependencies.class)
+@JsonDeserialize(as = ImmutableRecommendedProductDependencies.class)
+public interface RecommendedProductDependencies {
+
+    @JsonProperty("recommended-product-dependencies")
```

Comment:
things that are in config are generally kebab case, i also coppied "manifest-extensions" - no pref here though, can change